src/_includes/alert.njk

```diff
@@ -0,0 +1,9 @@
+
+{% macro alert(description, link, linktext) %}
+
+  <aside data-layout="alert" data-ccs-colors="dark">
```

Comment:
`data-ccs-colors` is used to apply a new color palette, switching modes would require `data-mode='dark'` -- but I think this looks fine as-is, so we can remove it:
 
```suggestion
  <aside data-layout="alert">
```